source/Calamari.Azure/Integration/AzurePowerShellScriptEngine.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Calamari.Deployment;
```

Comment:
The `AzurePowerShellScriptEngine', combined with the `ScriptEngineRegistry`, provided to way to plug-in the Azure script behaviour.  This will be able to be removed once the 3.0 Azure targets are retired.